project.clj

```diff
@@ -22,13 +22,23 @@
 
   :plugins [[lein-release "1.0.5"]]
 
-  :lein-release {:scm         :git
-                 :deploy-via  :lein-deploy}
+  :lein-release {:scm :git
+                 :deploy-via :lein-deploy}
```

Comment:
I think we actually don't need this boilerplate anymore; it was for the old lein-release plugin, and {{release}} is a task built in to lein itself now.  I think you can also get rid of the reference to the plugin above (and in any other repos you see this stuff still hanging around in).